learn/app-development/widgets/basic/search.md

```diff
@@ -53,6 +53,7 @@ For the Search widget you can set the following properties:
 | **Behavior** |
 | Read Only | Selecting this checkbox property prevents the user from being able to change the data value of a widget. |
 | Show | Showing determines whether or not a component is visible. It is a bindable property. |
+| Show clear | On setting show clear to true, the user is provided with an option to clear the inputted value in the search field. |
```

Comment:
Just say input value, instead of inputted. also "show clear" should be marked as code. Like this ```show clear```

Reply:
modified